ports/imagestriker/ImageStriker.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+get_controls
+
+# Variables
+GAMEDIR="/$directory/ports/imagestriker"
+GMLOADER_JSON="$GAMEDIR/gmloader.json"
+
+# CD and set permissions
+cd $GAMEDIR
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+$ESUDO chmod +x $GAMEDIR/gmloadernext.${DEVICE_ARCH}
+
+# Exports
+export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/lib:$LD_LIBRARY_PATH"
+export SDL_GAMECONTROLLERCONFIG="$sdl_controllerconfig"
+
+# Prepare game files
+if [ -f ./assets/data.win ]; then
+	mv assets/data.win assets/game.droid
```

Comment:
You put the game.droid inside the imagestriker.port right ? I just want to check that's what you want to do because that's not common. I think it works, but I'm not 200% sure.

Reply:
Yeah, I am not sure if you have missed the discussion on the discord, but from now on its agreed to put game files together with the runtime to go along with term 3 of the game maker runtime licence. https://gamemaker.io/en/legal/gamemaker-runtime-licence-free

Reply:
I might have missed the conclusion. So everything is fine then.